pkgs/by-name/ws/wstunnel-rust/package.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub
+, rustPlatform
+, lib
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wstunnel-rust";
```

Comment:
`pname` should be identical to upstream package name, and can be different from the attribute name, so `wstunnel` is fine.